app/coffeescript/browser-detect.coffee

```diff
@@ -30,6 +30,12 @@ define 'browserdetect', ->
       return if index == -1
       return parseFloat(dataString.substring(index + versionLabel.length + 1))
 
+    @isExplorer: ->
+      return BrowserDetect.platform().browser == 'explorer'
+
+    @isExplorer8: ->
+      return BrowserDetect.isExplorer() && BrowserDetect.platform().version == '8.0'
```

Comment:
i think we can replace all instances of `BrowserDetect` with `@` which would be more coffeesque